src/engraving/dom/durationelement.cpp

```diff
@@ -116,9 +116,16 @@ float DurationElement::timeStretchFactor() const
 //   actualTicks
 //---------------------------------------------------------
 
+Fraction DurationElement::actualTicks(Fraction& tick) const
```

Comment:
Also, what do you think about naming this method `actualTicksAt`? That might make its meaning slightly clearer, and also the meaning of the `tick` parameter.